tests/check-results.mk

```diff
@@ -0,0 +1,5 @@
+# Intended for inclusion by tests/*/Makefile.
```

Comment:
These definitions were the same in every makefile, with the exception of `tests/unit-fail-compilation/Makefile`. There, I simply override the one variable that is different.